src/applications/representative-appoint/components/SearchResult.jsx

```diff
@@ -0,0 +1,175 @@
+              >
+                <h3 aria-describedby={`representative-${representativeId}`}>
+                  {representativeName}
+                </h3>
+              </div>
+              {associatedOrgs?.length === 1 && (
+                <p style={{ marginTop: 0 }}>{associatedOrgs[0]}</p>
+              )}
+            </>
+          )}
+        </div>
+        {associatedOrgs?.length > 1 && (
+          <div className="associated-organizations-info vads-u-margin-top--1p5">
+            <va-additional-info
+              trigger="See associated organizations"
+              disable-border
+              uswds
+            >
+              {associatedOrgs?.map((org, index) => {
+                return (
+                  <>
+                    <p>{org}</p>
+                    {index < associatedOrgs.length - 1 ? (
+                      <br style={{ lineHeight: '0.625rem' }} />
+                    ) : null}
+                  </>
+                );
+              })}
+            </va-additional-info>
+          </div>
+        )}
+
+        <div className="representative-contact-section vads-u-margin-top--3">
+          {addressExists && (
+            <div className="address-link">
+              <a
+                href={`https://maps.google.com?saddr=${
+                  query?.context?.location
+                }&daddr=${address}`}
+                tabIndex="0"
+                className="address-anchor"
+                onClick={() => recordContactLinkClick()}
+                target="_blank"
+                rel="noreferrer"
+                aria-label={`${address} (opens in a new tab)`}
+              >
+                {addressLine1}{' '}
+                {addressLine2 ? (
+                  <>
+                    <br /> {addressLine2}
+                  </>
+                ) : null}{' '}
+                <br />
+                {city}, {stateCode} {zipCode}
+              </a>
+            </div>
+          )}
+          {phone && (
+            <div className="vads-u-margin-top--1p5">
+              {contact} {extension} {phone}
+              <va-telephone
+                contact={contact}
+                extension={extension}
+                onClick={() => recordContactLinkClick()}
+                disable-analytics
+              />
+            </div>
+          )}
+          {email && (
+            <div className="vads-u-margin-top--1p5">
+              <a
+                href={`mailto:${email}`}
+                onClick={() => recordContactLinkClick()}
+              >
+                {email}
+              </a>
+            </div>
+          )}
+        </div>
+      </div>
+    </va-card>
+  );
+};
+
+SearchResult.propTypes = {
+  addressLine1: PropTypes.string,
+  addressLine2: PropTypes.string,
+  addressLine3: PropTypes.string,
+  associatedOrgs: PropTypes.array,
+  city: PropTypes.string,
+  distance: PropTypes.string,
+  email: PropTypes.string,
+  key: PropTypes.number,
```

Comment:
`key` isn't listed as a prop in the function definition which causes a console error

Reply:
Removed.

Reply:
There's a reference to it on line 61 in `src/applications/representative-appoint/components/SelectAccreditedRepresentative.jsx`

Reply:
The `key` prop is unique and something that React references for render updates, so it can and should be passed to the child component - just not referenced within the child component itself.